spec/rails_helper.rb

```diff
@@ -177,8 +177,17 @@
       WebMock.disable_net_connect!
     else
       WebMock.disable_net_connect!(allow_localhost: true)
+      # Increase the default wait time for Capybara from 2 seconds to 5 seconds.
```

Comment:
I think we can remove these changes and keep wait_for_ajax, since other specs are using that as well

Reply:
You're right